docs/features/plugin/tools/index.mdx

```diff
@@ -1,187 +1,68 @@
-                # Note done is False here indicating we are still emitting statuses
-            }
-        )
-
-        # Do some other logic here
-        await __event_emitter__(
-            {
-                "type": "status",
-                "data": {"description": "Completed a task message", "done": True},
-                # Note done is True here indicating we are done emitting statuses
-            }
-        )
-
-        except Exception as e:
-            await __event_emitter__(
-                {
-                    "type": "status",
-                    "data": {"description": f"An error occured: {e}", "done": True},
-                }
-            )
-
-            return f"Tell the user: {e}"
-```
-</details>
-
-#### Message
-This type is used to append a message to the LLM at any stage in the Tool. This means that you can append messages, embed images, and even render web pages before, or after, or during the LLM response.
-
-```
-await __event_emitter__(
-                    {
-                        "type": "message", # We set the type here
-                        "data": {"content": "This message will be appended to the chat."},
-                        # Note that with message types we do NOT have to set a done condition
-                    }
-                )
-```
-
-<details>
-<summary>Example</summary>
-
-```
-async def test_function(
-        self, prompt: str, __user__: dict, __event_emitter__=None
-    ) -> str:
-        """
-        This is a demo
-
-        :param test: this is a test parameter
-        """
-
-        await __event_emitter__(
-                    {
-                        "type": "message", # We set the type here
-                        "data": {"content": "This message will be appended to the chat."},
-                        # Note that with message types we do NOT have to set a done condition
-                    }
-                )
-
-        except Exception as e:
-            await __event_emitter__(
-                {
-                    "type": "status",
-                    "data": {"description": f"An error occured: {e}", "done": True},
-                }
-            )
-
-            return f"Tell the user: {e}"
-```
-</details>
+Imagine you’re chatting with an LLM and want it to give you the current weather 🌦️. Normally, an LLM wouldn’t have access to live data, but with a **weather tool** enabled, the LLM can fetch and display this real-time information in the chat.
+
+**Tools** make this possible by allowing the LLM to **call external functions** and **retrieve relevant information** or **perform specific actions** during a conversation.
+
+#### 📝 Examples of Tools:
+
+1. **Web Search** 🔍: Get real-time answers from live web searches.
+2. **Image Generation** 🖼️: Create images based on user prompts.
+3. **Voice Synthesis** 🎙️: Integrate with an API like ElevenLabs to generate audio.
+
+## 🔑 How to Enable and Use Tools
```

Comment:
There are more ways to use tools.

Reply:
These are examples, how do you suggest changing them?

Reply:
Mention admins have access to all tools by default, and the following is needed for users with "user" role to access the tools.

Reply:
I do not understand that. What is the following? 

and you mean to add it as an example of tools?

Reply:
`🔑 How to Enable and Use Tools` section